saleor/order/emails.py

```diff
@@ -43,6 +45,38 @@ def collect_data_for_email(order_pk, template):
     }
 
 
+def collect_staff_data_for_email(order_pk, template):
```

Comment:
I would suggest renaming to `collect_staff_order_notification_data` - it tells more about what it does.